EDEngineer/Resources/Data/localization.json

```diff
@@ -1952,144 +1952,6 @@
       "de": "Bestandteil in keinem bekannten Bauplan!",
       "fr": null,
       "ru": null
-    },

```

Comment:
you need to fix the file by putting back what you removed ; I'll accept the PR after :)